segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/Storage.java

```diff
@@ -196,6 +199,17 @@
      */
     CompletableFuture<Iterator<SegmentProperties>> listSegments();
 
+    /**
```

Comment:
I don't think we should add new admin methods on storage interface.

The idea behind UtilsWrapper is that whatever admin commands we have, we add then to UtilsWrapper  and not to storage.

So in StreamSegmentContainer you do something like this

```
public CompletableFuture<List<ExtendedChunkInfo>> getExtendedChunkInfo(String streamSegmentName, Duration timeout) {
       val chunkedSegmentStorage = (ChunkedSegmentStorage) storage;
        UtilsWrapper wrapper = new UtilsWrapper(chunkedSegmentStorage, bufferSize, timeout);
        return wrapper.getExtendedChunkInfoList(streamSegmentName, true);
```

Reply:
Alternatively we move `UtilsWrapper::getExtendedChunkInfoList` to ChunkedSegmentStorage itself. However I am not comfortable adding new admin methods to main storage implementation. 

Similarly we should segregate admin functionality from normal segment container functionality.

Reply:
I've removed the addition on the `Storage` interface.